minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -263,57 +261,25 @@ public void initConfiguration() throws IOException {
     setAlluxioWorkDirectory();
     setHostname();
 
+    for (Entry<PropertyKey, String> entry : ConfigurationTestUtils.testConfigurationDefaults()
+        .entrySet()) {
+      Configuration.set(entry.getKey(), entry.getValue());
+    }
+
     Configuration.set(PropertyKey.TEST_MODE, true);
     Configuration.set(PropertyKey.WORK_DIR, mWorkDirectory);
-    Configuration.set(PropertyKey.USER_BLOCK_SIZE_BYTES_DEFAULT, DEFAULT_BLOCK_SIZE_BYTES);
```

Comment:
How do we decide what goes here vs the conf method?

Reply:
Properties specific to the `AbstractLocalAlluxioCluster` stay here, everything else moves to the conf method to be shared by other forms of test cluster